lib/spack/spack/compilers/clang.py

```diff
@@ -47,6 +49,32 @@ class Clang(Compiler):
                    'f77' : 'f77',
                    'fc'  : 'f90' }
 
+    @property
+    def is_apple(self):
+        ver_string = '%s' % self.version
+        if ver_string.endswith('-apple'):
+            return True
+        else:
+            return False
```

Comment:
Actually, this would be even better:
```python
@property
def is_apple(self):
    ver_string = str(self.version)
    return ver_string.endswith('-apple')
```